README.md

```diff
@@ -35,7 +35,7 @@ This project is built using _Maven_.
 
 ## Requirements
 
-1. Maven 3.5.0 or later.
+1. Maven 3.6.0 or later.
```

Comment:
I found Maven 3.5.0 does not work for the enforcer rule because of the dependency conflict #1361